tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -1131,3 +1148,536 @@ def record_stream(tensor: torch.Tensor, stream_id: int) -> None:
+
+    def get_valid_tactics(
+            self,
+            inputs: List[torch.Tensor],
+            profile: OptimizationProfile,
+            use_2cta_instrs: bool = False,
+            mma_tiler_mn: Tuple[int, int] = (128, 128),
+            cluster_shape_mn: Tuple[int, int] = (1, 1),
+            **kwargs,
+    ) -> List[int]:
+        # [b, m, k]
+        l, m, k = inputs[0].shape[0], inputs[0].shape[1], inputs[0].shape[2]
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)